action.yml

```diff
@@ -39,17 +42,20 @@ inputs:
   env_url:
     required: false
     description: The environment URL. (For `finish` only)
+  deployments:
+    required: false
+    description: Array of the deployments generated by github. (For `finish`)
```

Comment:
```suggestion
    description: Array of the deployments generated by GitHub. (For `finish`)
```

Reply:
Are these deployment IDs or names? Could you link to documentation in comments here?

I have a similar concern here as well, since it seems the interplay between `deployments` and `deployment_id` is unclear. Could we overload the existing option to accept arrays instead?

Reply:
I merge `deployment_id` and `deployments` properties into one property `deployment_id`. Unfortunately, the objects sent are differents. `deployment_id` is a string if `is_multi` is `false` or `undefined` whereas `deployment_id` is an array if `is_multi` is `true`

I updated the documentation.

Reply:
It would be more ergonomic if the type could be inferred, instead of relying on `is_multi` - e.g. if I pass in a comma-delimited string, the action should detect this as multiple deployments